test/test_client.py

```diff
@@ -27,6 +27,12 @@ def test_login(self):
         client.login(self.credentials.login, self.credentials.password, self.steam_guard_file)
         self.assertTrue(client.isLoggedIn)
 
+    def test_sessionid_cookie(self):
```

Comment:
Very nice test! Could you add another testing for the case you repaired. Somehow i made all the test and it didnt catch the missing sessionid cookie. Could you add the test that is deleting sessionid and failing for that reason with another request?